5calls/app/src/main/java/org/a5calls/android/a5calls/controller/StatsActivity.java

```diff
@@ -367,23 +358,23 @@ private Uri saveGraphImage() {
 
     private Bitmap generateGraphBitmap() {
         // Show a title for the share.
-        lineChart.setTitle("Calls over time");
+        binding.lineChart.setTitle("Calls over time");
```

Comment:
if you don't mind, can you put a TODO in here to make this a string in strings.xml? that way we can translate it properly and such. alternatively i should make an issue to audit for this...